test/integration/framework/goleak.go

```diff
@@ -24,74 +24,17 @@ import (
 )
 
 var (
+	// KnownGoroutineLeaks are leaks from just running KCP collected
+	// with TestGoleakWithDefaults and run through:
+	//	grep 'on top of the stack' output.log | cut -d, -f2- | cut -d' ' -f3 | sort | uniq
```

Comment:
Most of the leaks were just KCP not shutting down properly because it was stopped too fast after reporting ready, as noted in #3488 
The leftover are probably similar things as in the webhooks - goroutines or informer hooks that were started but never bound by a context.